### PR TITLE
fixes #93 - add statsd plugin (similar to graphite)

### LINKED ISSUE #93: Deploy statsd env variables
Similar to graphite, need these env variables for statsd:
* STATSD_PREFIX
* STATSD_HOSTNAME
* STATSD_PORT

Implementation detail, the prefix we use should NOT be the same as graphite publishing.  Maybe as simple as "lightblue" will be ok, since it will be embedded within the collectd/statsd prefix for the host that exists already.

### PR DESCRIPTION
**Pull request by dcrissman**
Essentially a duplication of the graphite puppet file and spec tests.

### CODE REVIEW

File: manifests/service/plugin/statsd.pp [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# == Class: lightblue::service::graphite
+#
+# Deploys/Removes lightblue graphite environment variables.
```
Comment (jewzaam) [inline manifests/service/plugin/statsd.pp]: s/graphite/statsd/g

File: spec/classes/lightblue/service/plugin/statsd_spec.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        }
+      end
+      
+      it do
+        should contain_file('/etc/profile.d/lb-statsd.sh') \
+        .with_content(/^export STATSD_PREFIX=#{statsd_prefix}$/)
+      end
+    end
+  end
+
```
Comment (jewzaam) [inline spec/classes/lightblue/service/plugin/statsd_spec.rb]: How about a test that the env variables are removed?  Wondering if you could setup a chain of puppet runs to test it.

File: manifests/service/plugin/statsd.pp (hunk 1 above)
Comment (dcrissman) [inline manifests/service/plugin/statsd.pp]: Thanks. Good catch.

File: spec/classes/lightblue/service/plugin/statsd_spec.rb (hunk 2 above)
Comment (dcrissman) [inline spec/classes/lightblue/service/plugin/statsd_spec.rb]: As it is, it wouldn't remove the env vars from currently open shells, but any new ones created would simply never have them set. I am not sure that there is a way to remove env vars from existing shells.

Comment (jewzaam) [inline spec/classes/lightblue/service/plugin/statsd_spec.rb]: Sorry, was intending to ask for a test that the file is removed from profile.d correctly.

Comment (dcrissman) [inline spec/classes/lightblue/service/plugin/statsd_spec.rb]: Oh, I see what you are saying. That is effectively what "statsd.sh is absent" is doing. I don't believe there is a way to chain tests together, but as Puppet ensures the final state will look a certain way, it is effectively the same thing.

**Pull request merged**